### DISCUSSION TITLE
Just hey 👋

### DISCUSSION THREAD

**Discussion by ishaga-bello** [community/community#20812] [Discussions]
Hey guys, hope you are doing gooood. My name is Bello and I'm from Cameroon 🇨🇲.
I love python and automations I intend to improve my skills and build awesome stuffs 🤖.
This was just to say hi to everyone and have a nice week 👍

**Comment by melvincwng** [comment]
Hey @ishaga-bello, welcome to GitHub Community 👋🏻 

Let's connect on GitHub, and I hope we can collaborate together in the future 😄 

If you are interested in contributing to a beginner-friendly open source repository which I've created, you can check it out over here - https://github.com/Your-First-Open-Source-Project

Have a good day ahead ^^

**Comment by dawid-szewc** [comment]
Hi

**Comment by Abrahami2** [comment]
Hi @ishaga-bello 
Keep at it! The community's always here to support. 🚀👍

**Comment by davidayoai** [comment]
Hello
let's connect and have fun together.
Check my AI tools here - [Davidayo Google Gemini](https://www.davidayo.com/artificial-intelligence/)

**Comment by edwingeorgeshaji** [comment]
Heyyy Ishagaa, nice meeting u...
Welcome to the community...

**Comment by DevKyLeOfficial** [comment]
Hello @ishaga-bello  Welcome, I do hope you enjoy coding like me as a newbie and still looking my comfort zone in coding and learning new programming languages

[a comment by an automated account is not shown]
